hdr_html_canvas_element.md

```diff
@@ -285,25 +285,49 @@ if the color volume of the image is smaller than that of the display.
 
 ## Add display color volume information to `Screen` interface of the CSS Object Model
 
-Add a new `displayColorVolume` attribute to the `Screen` interface:
+Add a new `screenColorInfo` attribute to the `Screen` interface:
 
 ```idl
 partial interface Screen {
-  optional ColorVolumeInfo displayColorVolume;
+  optional ScreenColorInfo screenColorInfo;
 }
 ```
 
-If present, the `displayColorVolume` attribute specifies the set of colors that
-the screen of the output device can reproduce without significant color volume
-mapping.
+```idl
+dictionary ScreenColorInfo {
+  optional ColorVolumeInfo colorVolume;
+  optional double referenceWhiteLuminance;
+}
+```
+
+If present,
+
+* `colorVolume` specifies, as defined above, the set of colors that the screen
+of the output device can reproduce without significant color volume mapping; and
+* `referenceWhiteLuminance` specifies the luminance of reference white as
+reproduced by the screen of the output device, and reflects viewing environment
+and user settings.
```

Comment:
From a whole-system kind-of view there are a few different ways to implement this...

1. The source targets a reference display and a reference viewing environment. The compositor works with the reference conditions. The sink itself is responsible for converting to the actual conditions, possibly with the help of a human changing settings on the sink.
2. The source targets a reference display and a reference viewing environment. The compositor is responsible for converting from the reference conditions to the actual conditions. The sink shows the signal as-is (ideally).
3. The source targets the actual conditions directly. The compositor does not change the signal. The sink shows the signal as-is.


Your sentence works just fine for all cases. In the first two, all the `ScreenColorInfo` values would be fixed all the time and the display/user will do the adjustment. In the second case the values will be dynamic.

While this probably shouldn't be part of the spec, we really should make sure that it's understood that those values do not necessarily imply real-world luminances and real-world display color volume.

Not making it clear has already caused a generation of desktop displays not having adjustable HDR modes.

Reply:
> While this probably shouldn't be part of the spec, we really should make sure that it's understood that those values do not necessarily imply real-world luminances and real-world display color volume.

I have tried writing such a sentence a couple of times, and have not found the right words. Maybe something as informal and short as you suggest above would work.